content/en/academy/localstack-developement/lesson-2/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Deploy a full fledged containerised application using LocalStack"
+linkTitle: "Deploy a full fledged containerised application using LocalStack"
+weight: 2
+description: >
+  In this video we will do a walkthrough of the web application we are going to deploy. This application mimics a real world application’s complexity to some extent. We will try adding data to the dynamodb through the web application and then query the same data from the Localstack resource browser. We will see how Localstack has improved the developer experience of cloud applications. Post this we will discuss the architecture diagram and explain how the whole application flow goes through each of these AWS services. Towards the end we will try to manually create a few resources for the project, such as DynamoDB table.
```

Comment:
Words like `dynamodb`, `iam`, `terraform`, `localstack` need to be properly cased across all lessons